unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Quest_V2/QuestCompletedHUD/IQuestCompletedComponentView.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+
+namespace DCL.Quest
```

Comment:
i think that the namespace should be in plural, to avoid possible conflicts with a class name `Quest`

Reply:
done